kratos/integration/integration_point_utilities.cpp

```diff
@@ -15,6 +15,33 @@
 
 namespace Kratos
 {
+    /* @brief Creates integration points within provided spans according
+     *        to the provided points per span.
+     * @param rIntegrationPoints result integration points.
+     * @param rSpansLocalSpace spans in which integration points are generated.
+     * @param IntegrationPointsPerSpan number of integration points per span.
+     */
+    void IntegrationPointUtilities::CreateIntegrationPoints1D(
+        IntegrationPointsArrayType& rIntegrationPoints,
+        const std::vector<double>& rSpansLocalSpace,
+        SizeType IntegrationPointsPerSpan)
```

Comment:
can't `IntegrationPointsPerSpan` be const?

Reply:
Hmm good point. I quite remember that it wasn't more efficient.

@philbucher integer and size types were ideally passed without reference nor const, right?

Reply:
afaik (@roigcarlo please correct me if I am wrong)
`int`: 32bit
`std::size_t`: 64bit
`int&`: 64bit
`std::size_t&`: 64bit

const or not is up to you => what do you intend? Should this value be mutable? If no, then make it const

Reply:
Thanks for the explanation. 👍 

It's not touched do it'd be better to make it const then.

Reply:
sweet. it is const now.